.circleci/config.yml

```diff
@@ -927,7 +927,7 @@ jobs:
             # Use make for bash-like environment variable substitution
 
             # Codesign release (tagged) builds only
-            If ( $env:CIRCLE_TAG ) { $env:ARG1='-c.win.certificateSubjectName=""Automattic, Inc.""' }
+            If ( $env:CIRCLE_TAG ) { $env:ARG1='-c.win.certificateSubjectName="Automattic, Inc."' }
```

Comment:
@nsakaimbo this fixes the CI issue as shown in https://github.com/Automattic/wp-calypso/pull/49903.

I snuck the commit straight in here without going through the full release process because it's such a small thing.